templates/repo/issue/view_content/pull_merge_instruction.tmpl

```diff
@@ -5,6 +5,7 @@
 	<div class="ui secondary segment">
 		{{if eq $.Issue.PullRequest.Flow 0}}
 		<div>git checkout -b {{if ne $.Issue.PullRequest.HeadRepo.ID $.Issue.PullRequest.BaseRepo.ID}}{{$.Issue.PullRequest.HeadRepo.OwnerName}}-{{end}}{{$.Issue.PullRequest.HeadBranch}} {{$.Issue.PullRequest.BaseBranch}}</div>
+		{{/* the only legacy HTMLURL used in template, which doesn't affect users too much and is very diffcult to fix */}}
```

Comment:
It is possible to fix that?
I would have thought that it's absolutely necessary for all cloning instructions…

Reply:
It's not easy at the moment.

If there would be a clear solution, I would have fixed it ..... 😭 

So just keep it there for the future changes.

Reply:
FYI: There are still many calls to `AppUrl` from the template, for example, the package help manual.


They are the similar problems, and could be fixed in the future together.

Reply:
I get a good idea: use AppUrl to replace HTMLURL, then this problem is covered to the AppUrl problem now. See  https://github.com/go-gitea/gitea/pull/22831/commits/06fff8768d7be6eff0f28e820f097b7bf4aedd9c